app/scripts/directives/podMetrics.js

```diff
@@ -358,6 +358,9 @@ angular.module('openshiftConsole')
               // get the unit value if specified
               var unit =  metric.unit || "";
 
+              // we only want part of the metric ID - "custom/" and everything after it
+              var datasetId = "custom/" + metric.id.replace(/.*\/custom\//, '');
```

Comment:
@jmazzitelli Do you have an example of the metrics ID? It would be good to give an example in the comment as well.

cc @mwringe